src/nni_engine.hpp

```diff
@@ -237,15 +332,15 @@ class NNIEngine {
   double GetScoreForNNI(const NNIOperation &nni) const;
 
   // Update adjacent NNIs at end of current sweep. If not re-evaluating rejected NNIs,
-  // then current adjacent NNIs are removed. Then NNIs that are adjacent to last sweep's
-  // accepted NNIs are added.
-  void UpdateAdjacentNNIs(const bool reevaluate_rejected_nnis = false);
+  // then current adjacent NNIs are removed. Then NNIs that are adjacent to last
+  // sweep's accepted NNIs are added.
+  void UpdateAdjacentNNIs(std::optional<bool> reevaluate_rejected_nnis = std::nullopt);
```

Comment:
let's keep it simple

Reply:
Removed.  Option now set though class member.